protected/humhub/modules/stream/actions/Stream.php

```diff
@@ -295,17 +295,26 @@ protected function isInitialRequest()
      */
     public static function renderEntry(ContentActiveRecord $record, $options =  [], $partial = true)
     {
+        // TODO should be removed in next major version
         // Compatibility with pre 1.2.2
         if(is_bool($options)) {
             $partial = $options;
             $options = [];
         }
 
```

Comment:
How about just adding

```
if(!$record->wallEntryClass) {
 return "";
}
```

And in v1.3 we should get rid of the static `@humhub/modules/content/views/layouts/wallEntry` and integrate the div into `@humhub/modules/content/widgets/views/wallEntry.php`, so the WallEntry widget is more flexible and not dependend on the `$jsWidget` etc.